snuba/datasets/entities/events.py

```diff
@@ -60,7 +60,7 @@ def select_storage(
         return StorageAndMappers(storage, self.__mappers)
 
 
-class EventsEntity(Entity):
+class BaseEventsEntity(Entity):
```

Comment:
Could youo please make this abstract by inheriting ABC? So only subclasses can be initialized.